__tests__/app.test.js

```diff
@@ -321,10 +325,55 @@ describe('/api/articles/:article_id/comments', () => {
       .expect(200)
       .then((res) => {
         res.body.forEach((user) => {
+          expect(res.body.length > 0).toBe(true)
           expect(user).hasOwnProperty("username");
           expect(user).hasOwnProperty("name");
           expect(user).hasOwnProperty("avatar_url");
         })
       })
     })
-  });
\ No newline at end of file
+  });
+
+  describe('GET /api/articles (topic query)', () => {
+    test('Returns an array of article objects with only the topics of mitch',() => {
+      return request(app)
+      .get('/api/articles/?topic=mitch')
```

Comment:
The endpoint should look like this instead:
`/api/articles?topic=mitch
`
No forward slash before the question mark.
Please refactor this is all tests.